projects/valtimo/dossier/src/lib/components/dossier-detail/tab/progress/progress.component.html

```diff
@@ -14,31 +14,52 @@
   ~ limitations under the License.
   -->
 
-<div
+<ng-container
   class="full-height-tab-content"
-  *ngIf="
-    processDocumentInstances && processDocumentInstances.length > 0;
-    else emptyProcessDocumentInstances
-  "
+  *ngIf="{
+    processInstanceItems: processInstanceItems$ | async,
+    selectedProcessInstance: selectedProcessInstance$ | async
+  } as obs"
 >
-  <div class="col-3">
-    <label><strong>Process</strong></label
-    ><br />
-    <select class="form-control" (change)="loadProcessInstance($event.target.value)">
-      <option
-        *ngFor="let processDocumentInstance of processDocumentInstances"
-        [value]="processDocumentInstance.id.processInstanceId"
-        [selected]="selectedProcessInstanceId === processDocumentInstance.id.processInstanceId"
+  <div *ngIf="!obs.processInstanceItems">
+    <span>{{ 'progress.noProcessDocumentInstances' | translate }}</span>
+  </div>
+  <div class="process-info-row">
+    <div *ngIf="obs.processInstanceItems">
```

Comment:
Add the ngIf directly on the combo-box and remove the div

Reply:
Changed